### PR TITLE
Tethering agent: fix peer URI construction

### PR DESCRIPTION
**Pull request by rmstar**

### CODE REVIEW

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java [hunk 1]
```diff
@@ -90,7 +90,8 @@
     .registerTypeAdapter(Arguments.class, new ArgumentsCodec())
     .registerTypeAdapter(ProgramOptions.class, new ProgramOptionsCodec())
     .create();
-  private static final String CONNECT_CONTROL_CHANNEL = "/v3/tethering/controlchannels/";
+  static final String CREATE_TETHER = "v3/tethering/connections/";
```
Comment (nliu132) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java]: nit: private variable

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java [hunk 2]
```diff
@@ -198,6 +199,11 @@ protected long runTask() {
     return connectionInterval;
   }
 
+  private URI appendPath(URI baseUri, String path) {
```
Comment (nliu132) [inline cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java]: is there a benefit to creating the base URI and resolving `newPath` against it, vs having this method take `(String base, String path)` and create the return URI from `newPath`?

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java (hunk 1 above)
Comment (rmstar) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java]: fixed

File: cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java (hunk 2 above)
Comment (rmstar) [reply cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java]: fixed to return new path from this method. URI resolving is done by the caller

**Review by nliu132 — APPROVED** [review]

**Pull request merged**